setup/env_file.yml

```diff
@@ -274,5 +274,5 @@ dependencies:
     - pyqtgraph==0.10.0
     - pyvisa==1.10.1
     - retrying==1.3.3
-    - rpyc==4.1.1
```

Comment:
is there a reason why you removed rpyc?

Reply:
Good catch, this was an accident. Fixed in 34a23155a7a4de7707c3bdb00590f5a33f36ba63.